resources/assets/js/tabler.js

```diff
@@ -1,18 +1,14 @@
-// Sidebar layout - Highlight active section/page
-$('aside a[href="'+window.location.href+'"]')
-    .addClass('active')
-    .parent()
-    .addClass('show')
-    .prev()
-    .addClass('show');
-
-// Topbar layout - Highlight active section/page
-$('header.top a[href="'+window.location.href+'"]')
-    .addClass('active')
-    .parent()
-    .addClass('active')
-    .parent()
-    .addClass('active');
-
+// Set menu auto close state
 document.querySelectorAll('header a.nav-link.dropdown-toggle').forEach(el => el.dataset.bsAutoClose = 'true');
-document.querySelectorAll('aside a.nav-link.dropdown-toggle').forEach(el => el.dataset.bsAutoClose = 'false');
\ No newline at end of file
+document.querySelectorAll('aside a.nav-link.dropdown-toggle').forEach(el => el.dataset.bsAutoClose = 'false');
+
+// Bar layout - Highlight active section/page
+[...document.querySelectorAll('aside a[href], header.top a[href]')]
+    .filter(el => window.location.href.startsWith(el.href) && !el.href.endsWith('#'))
+    .forEach(el => {
+        while (/(nav-item|nav-link|dropdown)/.test(el.className)) {
+            if (el.previousElementSibling?.dataset.bsAutoClose === 'false') el.classList.add('show');
+            el.classList.add('active');
+            el = el.parentElement;
+        }
+    });
```

Comment:
Previous code was not working, it was not setting the active side/top bar item.
Now it is, even if the item is a "inner" item of a menu, like an `article/1/edit`, it is selecting the `articles` menu item.

![image](https://github.com/Laravel-Backpack/theme-tabler/assets/1838187/f44959f9-f470-49fa-acc7-1d4482e735b3)